templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -295,12 +295,13 @@
     "nuclear_exclude = True\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        nuc_seg_suffix="nuclear.tiff",
</pre><p>Thanks! I think this is the last change - <code>nuc_seg_suffix</code> should be <code>"_nuclear.tiff"</code> here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>